videointelligence/setup.py

```diff
@@ -39,7 +39,9 @@
     packages=find_packages(exclude=('tests*',)),
     install_requires=(
         'googleapis-common-protos >= 1.5.3, < 2.0dev',
+        # GAX remains for the time being for the v1beta1 interface.
```

Comment:
We can (and probably should) regen the v1beta1 interface.

Reply:
It *does* drop `options` in favor of `retry` and `timeout` on every method and changes the constructor significantly.